ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-eus-4.10.yaml

```diff
@@ -0,0 +1,83 @@
+base_images:
+  ansible:
+    name: "4.10"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.10"
+  tools:
+    name: "4.10"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.11"
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  target:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-ovn-p2-f30
+  cron: 26 1 24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_INTERMEDIATE_TAGS: '@4.11 and @upgrade-sanity and @aws-ipi and @network-openshiftsdn
+        and not @fips'
+      E2E_RUN_LATEST_TAGS: '@4.12 and @upgrade-sanity and @aws-ipi and @network-openshiftsdn
+        and not @fips'
+      UPGRADE_CHECK_RUN_TAGS: '@4.12 and @upgrade-check and @aws-ipi and @network-openshiftsdn
+        and not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@4.10 and @upgrade-prepare and @aws-ipi and @network-openshiftsdn
```

Comment:
network-openshiftsdn -> network-ovnkubernetes

Reply:
Good catch! Updated. Thank you.